lib/jsonapi_parameters/translator.rb

```diff
@@ -60,22 +62,41 @@ def jsonapi_relationships
   end
 
   def handle_to_many_relation(relationship_key, relationship_value)
-    key = "#{relationship_key.to_s.pluralize}_attributes".to_sym
+    with_inclusion = true
 
-    val = relationship_value.map do |relationship_value|
+    vals = relationship_value.map do |relationship_value|
       related_id = relationship_value.dig(:id)
       related_type = relationship_value.dig(:type)
 
       included_object = find_included_object(
         related_id: related_id, related_type: related_type
       ) || {}
 
-      included_object.delete(:type)
+      # If at least one related object has not been found in `included` tree,
```

Comment:
We may consider giving some example of that in the documentation too. It can be checked by a developer in the code but for a newbies in JSON:Api on Rails world it may not be so clear how it works. What do you think about it?